src/main/java/com/buschmais/jqassistant/scm/maven/ExportDatabaseMojo.java

```diff
@@ -1,62 +0,0 @@
-package com.buschmais.jqassistant.scm.maven;
```

Comment:
Why did you remove this feature from the plugin?

Reply:
The Neo4j API for dumping to Cypher is no longer available.

Reply:
Ok, but we should mention this in the release notes document for the upcomming release.